modules/deployment-model-impl/src/test/java/org/mule/runtime/module/deployment/impl/internal/application/ApplicationDescriptorFactoryTestCase.java

```diff
@@ -91,6 +91,11 @@ private static File getResourceFile(String resource) {
                                                                         .getAbsolutePath());
 
   @Rule
+  public SystemProperty remoteMavenRepository =
+      new SystemProperty("muleRuntimeConfig.maven.repositories.mulesoftPublic.url",
```

Comment:
but why it has to download the dependencies instead of using the ones that are built on the same reactor?